src/vs/platform/extensionManagement/node/extensionGalleryService.ts

```diff
@@ -24,6 +24,7 @@ import { readFile } from 'vs/base/node/pfs';
 import { writeFileAndFlushSync } from 'vs/base/node/extfs';
 import { generateUuid, isUUID } from 'vs/base/common/uuid';
 import { values } from 'vs/base/common/map';
+import { forEach } from '../../../base/common/event';
```

Comment:
we should add `// {{SQL CARBON EDIT}}` tag to each edit in this file so we don't blow them away in the next merge

Reply:
I recommend using absolute paths for includes.  vs code moves there files around periodically.